pkg/models/system_intake.go

```diff
@@ -15,10 +15,12 @@ const (
 	SystemIntakeStatusDRAFT SystemIntakeStatus = "DRAFT"
 	// SystemIntakeStatusSUBMITTED captures enum value "SUBMITTED"
 	SystemIntakeStatusSUBMITTED SystemIntakeStatus = "SUBMITTED"
-	// SystemIntakeStatusREVIEWED captures enum value "REVIEWED"
-	SystemIntakeStatusREVIEWED SystemIntakeStatus = "REVIEWED"
-	// SystemIntakeStatusREJECTED captures enum value "REJECTED"
-	SystemIntakeStatusREJECTED SystemIntakeStatus = "REJECTED"
+	// SystemIntakeStatusACCEPTED captures enum value "ACCEPTED"
+	SystemIntakeStatusACCEPTED SystemIntakeStatus = "ACCEPTED"
```

Comment:
@Igarfinkle I think these statuses affect the work you're doing on the intake reviewer track. Can we confirm these status values are sync'd between your branches?